packages/@webex/internal-plugin-mercury/src/socket/socket-base.js

```diff
@@ -489,11 +489,21 @@ export default class Socket extends EventEmitter {
       }
     };
 
+    const calculateLatency = (pingTimestamp) => {
+      const now = new Date().getTime();
```

Comment:
wouldn't it be better to use `performance.now()`?

Reply:
added